src/services/SecurutyService.ts

```diff
@@ -1,16 +1,49 @@
+/* eslint-disable class-methods-use-this */
 import { UserInfo } from "../app/features/auth/models";
 
 const REFRESH_TOKEN = "refreshToken";
 const ACCESS_TOKEN = "accessToken";
 const USER_INFO = "userInfo";
 
+export class SecurityError extends Error {
+  constructor(message: string) {
+    super();
```

Comment:
anyways, i think you can call super(message) instead of setting message yourself

Reply:
According to [this source](https://learn.javascript.ru/oop-errors) parent Error constructor doesn't set message property.